azure-pipelines.yml

```diff
@@ -49,4 +49,33 @@ stages:
         targetPort: '3000'
         ingress: 'external'
         imageToDeploy: '$(containerRegistry)/$(imageRepository):$(tag)'
-        environmentVariables: 'APP_ENV=$(app_env) DB_HOST=$(db_host) DB_PORT=5432 DB_USERNAME=$(db_username) DB_PASSWORD=$(db_password) DB_DATABASE=$(db_database) AZURE_DATALAKE_STORAGE_ACCOUNT_NAME=$(azure_datalake_storage_account_name) AZURE_DATALAKE_STORAGE_ACCOUNT_KEY=$(azure_datalake_storage_account_key) AZURE_DATALAKE_STORAGE_DIRECTORY_NAME=$(azure_datalake_storage_directory_name) AZURE_BLOB_STORAGE_ACCOUNT_NAME=$(azure_blob_storage_account_name) AZURE_BLOB_STORAGE_ACCOUNT_KEY=$(azure_blob_storage_account_key) AZURE_BLOB_STORAGE_CONTAINER_NAME=$(azure_blob_storage_container_name) BACKEND_URL=$(backend_url) FRONTEND_URL=$(frontend_url) SESSION_SECRET=$(session_secret) SESSION_STORE=$(session_store) JWT_SECRET=$(jwt_secret) JWT_EXPIRES_IN=$(jwt_expires_in) AUTH_PROVIDERS=$(auth_providers) GOOGLE_CLIENT_ID=$(google_client_id) GOOGLE_CLIENT_SECRET=$(google_client_secret) ONELOGIN_URL=$(onelogin_url) ONELOGIN_CLIENT_ID=$(onelogin_client_id) ONELOGIN_CLIENT_SECRET=$(onelogin_client_secret) ONELOGIN_PUBLIC_KEY=$(onelogin_public_key) ONELOGIN_PRIVATE_KEY=$(onelogin_private_key) REDIS_URL=$(redis_url) REDIS_ACCESS_KEY=$(redis_access_key)'
+        environmentVariables: >
+          APP_ENV=$(app_env)
+          DB_HOST=$(db_host)
+          DB_PORT=5432
+          DB_USERNAME=$(db_username)
+          DB_PASSWORD=$(db_password)
+          DB_DATABASE=$(db_database)
+          AZURE_DATALAKE_STORAGE_ACCOUNT_NAME=$(azure_datalake_storage_account_name)
+          AZURE_DATALAKE_STORAGE_ACCOUNT_KEY=$(azure_datalake_storage_account_key)
+          AZURE_DATALAKE_STORAGE_DIRECTORY_NAME=$(azure_datalake_storage_directory_name)
+          AZURE_DATALAKE_STORAGE_FILESYSTEM_NAME=$(azure_datalake_storage_filesystem_name)
+          AZURE_BLOB_STORAGE_ACCOUNT_NAME=$(azure_blob_storage_account_name)
+          AZURE_BLOB_STORAGE_ACCOUNT_KEY=$(azure_blob_storage_account_key)
+          AZURE_BLOB_STORAGE_CONTAINER_NAME=$(azure_blob_storage_container_name)
+          BACKEND_URL=$(backend_url)
+          FRONTEND_URL=$(frontend_url)
+          SESSION_SECRET=$(session_secret)
+          SESSION_STORE=$(session_store)
+          JWT_SECRET=$(jwt_secret)
+          JWT_EXPIRES_IN=$(jwt_expires_in)
+          AUTH_PROVIDERS=$(auth_providers)
+          GOOGLE_CLIENT_ID=$(google_client_id)
+          GOOGLE_CLIENT_SECRET=$(google_client_secret)
+          ONELOGIN_URL=$(onelogin_url)
+          ONELOGIN_CLIENT_ID=$(onelogin_client_id)
+          ONELOGIN_CLIENT_SECRET=$(onelogin_client_secret)
+          ONELOGIN_PUBLIC_KEY=$(onelogin_public_key)
+          ONELOGIN_PRIVATE_KEY=$(onelogin_private_key)
+          REDIS_URL=$(redis_url)
+          REDIS_ACCESS_KEY=$(redis_access_key)
```

Comment:
This should return as one line without newlines, but will make the git diffs a lot easier to read.